### DISCUSSION TITLE
Using structured piano notes for different emotions

### DISCUSSION THREAD

**Discussion by anirban-oss** [monkeytypegame/monkeytype#4151] [Ideas]
Can we use certain music snippets that are played according to the mood of the text being written, specially for quotes, a sad music for sad quote and happy for respectively, we can start with just binary classification, i think we can leverage hugging face to classify and just add the tones to them?

**Comment by unrealapex** [comment]
This is an interesting idea, but it seems outside the scope of this project. Perhaps this could be its own website? There is sound on click(`Sound on click > pentatonic`) though.

**Reply by anirban-oss** [reply]
Okay, let me try this on my own by forking, it feels cool, thank you for the idea.
